versions.yaml

```diff
@@ -134,13 +134,13 @@ module-sets:
       - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dotnetdiagnosticsreceiver
       - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/elasticsearchreceiver
       - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver
-      - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver 
-      - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudpubsubreceiver 
-      - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudspannerreceiver 
-      - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver 
+      - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver
```

Comment:
Removing these trailing spaces makes sense, but they are not related to the PR. Could you please remove them? Feel free to submit another PR if you want to

Reply:
removed